### PR TITLE
Introduce tool template

### PR DESCRIPTION
**Pull request by rahulku**
> Please add a description of your PR.
> If this is a solution to an open challenge, please explain your solution.
>
> Don't forget to check our book to ensure your solution satisfy the overall
> requirements as well as the challenge success criteria.
>

Resolves #ISSUE-NUMBER

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 and MIT licenses.

### CODE REVIEW

File: .github/TOOL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+### Licenses
+<!-- Please list the licenses that the tool uses and can be used under. -->
+
+### Steps to Use the Tool
+
+1. [First Step]
+2. [Second Step]
+3. [and so on...]
+
+### Artifacts
```
Comment (jaisnan) [inline .github/TOOL_REQUEST_TEMPLATE.md:27]: non-blocking: I'd separate this section into two 
1. ### tutorial
 _Include an example of using the tool to perform verification _

2. ### artifacts
_Include links, papers, blog posts about your tool_

Comment (rahulku) [reply .github/TOOL_REQUEST_TEMPLATE.md:27]: my bad. I actually missed a hashtag. fixing.

Comment (rahulku) [reply .github/TOOL_REQUEST_TEMPLATE.md]: fixed.

**Review by jaisnan — APPROVED** [review]
lgtm! Can refine and make modifications as we go

**Pull request merged**